src/main.js

```diff
@@ -115,6 +115,7 @@ const startWithInputFile = async (inputFile) => {
         styles: 'styles',
         mbtiles: inputFilePath,
         pmtiles: inputFilePath,
+        files: inputFilePath
```

Comment:
I would just remove this line. It only gets used if a single mbtiles/pmtiles is specfied and in that case, it doesn't really need this files config

Reply:
In fact I'm using exactly this setup for testing (using the docker image with a slightly modified basic-preview style + added geojson)  ;-)